bindings/wallet-js/src/lib.rs

```diff
@@ -340,15 +387,17 @@ macro_rules! impl_secret_key {
 
         #[wasm_bindgen]
         impl $name {
+            /// Generates the key using OS-provided entropy.
```

Comment:
Maybe it's worth pointing out to the specific apis that `OsRng` uses? given that this is a wasm/js specific crate they should be only browser and node.
Not really sure if a web developer would find that useful though.